### PR TITLE
Install sql files update

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #15564 .

### Summary of Changes
Make sure that all the default options for com_users, com_contacts and com_content are set in the database at installation. 
It didnt use to be that important but now that we read the default values to show what the global option is it is more important that they are set in the database without having to ask the user to save their options even if they havent changed anything

### Testing Instructions
Perform a full install with this branch

### CODE REVIEW

**Comment by franz-wohlkoenig** [comment]
are `Basic Settings` meant as default Options for com_users? If yes, they are not shown:
bildschirmfoto 2017-05-20 um 12 19 32

**Comment by tonypartridge** [comment]
@franz-wohlkoenig how do you mean? It's set to use default. Brian's change sets the default values, this is use default for instance. I.e. the Actively set backend template.

**Comment by tonypartridge** [comment]
@brianteeman I've tested / reviewed this, what about date_format should we specify a default data_format value? i.e. D M Y as the value so users know how to use it (Without reading the tooltip) ;-)

**Comment by franz-wohlkoenig** [comment]
@tonypartridge reading brians, yours and my Comment – didn't get the point of all 3. So it looks like a lack of langugage.

**Comment by brianteeman** [comment]
@tonypartridge i created this by comparing the results of the db value after saving the options once.

**Comment by tonypartridge** [comment]
Ok that's great then, we can leave as is.

**Comment by tonypartridge** [comment]
I have tested this item :white_check_mark: successfully on 6520701b43b4e21339d2ff47fc989104eee5ae3b<br /><br />Tested as a fresh install and compared. values before and after.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16138">issues.joomla.org/tracker/joomla-cms/16138</a>.</sub>

File: installation/sql/mysql/joomla.sql [hunk 1]
```diff
@@ -492,7 +492,7 @@ INSERT INTO `#__extensions` (`extension_id`, `package_id`, `name`, `type`, `elem
...
...s":"0","linka_name":"","linkb_name":"","linkc_name":"","linkd_name":"","linke_name":"","contact_icons":"0","icon_address":"","icon_email":"","icon_telephone":"","icon_mobile":"","icon_fax":"","icon_misc":"","category_layout":"_:default","show_category_title":"1","show_description":"1","show_description_image":"0","maxLevel":"-1","show_subcat_desc":"1","show_empty_categories":"0","show_cat_items":"1","show_cat_tags":"1","show_base_description":"1","maxLevelcat":"-1","show_subcat_desc_cat":"1","show_empty_categories_cat":"0","show_cat_items_cat":"1","filter_field":"0","show_pagination_limit":"0","show_headings":"1","show_image_heading":"0","show_position_headings":"1","show_email_headings":"0","show_telephone_headings":"1","show_mobile_headings":"0","show_fax_headings":"0","show_suburb_headings":"1","show_state_headings":"1","show_country_headings":"1","show_pagination":"2","show_pagination_results":"1","initial_sort":"ordering","captcha":"","show_email_form":"1","show_email_copy":"1","banned_email":"","banned_subject":"","banned_text":"","validate_session":"1","custom_reply":"0","redirect":"","show_feed_link":"1","custom_fields_enable":"1"}', '', '', 0, '0000-00-00 00:00:00', 0, 0),
```
Comment (Quy) [inline installation/sql/mysql/joomla.sql]: From PR#15973: `"add_mailto_link":"0"`

**Comment by brianteeman** [comment]
@quy applied that new param
can you and @tonypartridge retest please

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 49851d0a57ac47ef1ffda4a3120029c241373953<br /><br />Code review and tested with MySQL installation.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16138">issues.joomla.org/tracker/joomla-cms/16138</a>.</sub>

**Pull request closed without merging**